src/models/MarkdownFile.ts

```diff
@@ -0,0 +1,111 @@
+import { Uri, workspace, window, Position } from 'vscode';
+import { EOL } from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { Card } from './Card';
+
+
+export class MarkdownFile {
+  private readonly metaStart = `[//]: # (Autogenerated Anki Metadata -- delete to disable sync)`;
```

Comment:
This is quite verbose and will be on a lot of cards so would be nice if this is as short as possible.
I think "[Anki Metadata], [End Anki Metadata]" is fine, we can add to the `README` that you can delete it to disable sync.